### DISCUSSION TITLE
Exercise 5:  Label with proper vector clock all the events of the distributed computation in image vector.pdf.

### DISCUSSION THREAD

**Discussion by dag7dev** [sapienzastudentsnetwork/distributed-systems#30] [Exercises]
![image](https://github.com/sapienzastudentsnetwork/distributed-system-2023-2024/assets/44711271/77649ca7-f681-4f53-9deb-7915d309ea61)

**Comment by susannacifani** [comment]
Vector clock (the image is in Exercise 1, on Classroom): 
![image](https://github.com/sapienzastudentsnetwork/distributed-system-2023-2024/assets/73530772/4cb69090-df4f-41f5-8978-10baeaf24dec)

**Comment by dag7dev** [comment]
> Vector clock (the image is in Exercise 1, on Classroom): ![image](https://user-images.githubusercontent.com/73530772/277644383-4cb69090-df4f-41f5-8978-10baeaf24dec.jpeg)

Thanks, added!
